Uninitialized field warning with clang using OE_SET_ENCLAVE_SGX

```
props.cc:7:1: error: missing field 'oeinfo_size' initializer [-Werror,-Wmissing-field-initializers]
OE_SET_ENCLAVE_SGX(
^
/opt/openenclave/include/openenclave/bits/sgx/sgxproperties.h:132:9: note: expanded from macro 'OE_SET_ENCLAVE_SGX'
```
...when compiling with clang and `-Wall -Wextra -Werror`

Current work-around: `-Wno-error=missing-field-initializers`.

--- a/include/openenclave/bits/sgx/sgxproperties.h
+++ b/include/openenclave/bits/sgx/sgxproperties.h
@@ -128,7 +128,7 @@ typedef struct _oe_sgx_enclave_properties
         },                                                                \
         .image_info =                                                     \
         {                                                                 \
-            0                                                             \
+            0, 0, 0, 0, 0, 0                                              \
         },                                                                \
         .sigstruct =                                                      \
         {                                                                 \

